README.md

```diff
@@ -25,12 +25,22 @@ _see below for a simpler conceal-based solution_
 ```lua
 use {"akinsho/org-bullets.nvim", config = function()
   require("org-bullets").setup {
+    -- Symbols for different headline bullet levels
     symbols = { "◉", "○", "✸", "✿" }
-    -- or a function that receives the defaults and returns a list
+    -- Can also be overridden with a function to append to the above defaults
     symbols = function(default_list)
       table.insert(default_list, "♥")
       return default_list
-    end
+    end,
+    -- Show the literal text of the current line, similar to 'concealcursor'
```

Comment:
@levouh sorry for the very long wait to review this, so this option is not actually fit for public use, I never actually got it working properly so would be better to keep it hidden